src/internal/connector/graph/errors.go

```diff
@@ -36,6 +36,7 @@ const (
 	errCodeResourceNotFound            = "ResourceNotFound"
 	errCodeRequestResourceNotFound     = "Request_ResourceNotFound"
 	errCodeMailboxNotEnabledForRESTAPI = "MailboxNotEnabledForRESTAPI"
+	errCodeErrorAccessDenied           = "ErrorAccessDenied"
```

Comment:
Is this different than a 401 status code?

Reply:
access denied is 403.  401 is unauthorised.